model/src/main/scala/hmda/model/census/Census.scala

```diff
@@ -0,0 +1,65 @@
+package hmda.model.census
+
+object Census {
+  val states = Map(
```

Comment:
At the moment this is more complicated than it needs to be, as the state names aren't used. I could imagine there's a future purpose in mind; is there?